app/views/doctors/show.html.erb

```diff
@@ -0,0 +1,13 @@
+Name: <%= @doctor.name %><br>
+Specialty: <%= @doctor.specialty %><br>
+University: <%= @doctor.university %><br>
+Hospital: <%= @doctor.hospital.name %><br>
+
+<div id="patients">
+  <% @doctor.patients.each do |patient| %>
+    <div id="patient_id-<%= patient.id %>">
+      <%= patient.name %>
+      <%= button_to "Delete", doctor_patient_path(@doctor, patient), method: :delete %>
```

Comment:
This route helper path is not correctly being used. This route helper only takes in one ID, the ID of a `DoctorPatient` object, you are passing in two IDs, and the IDs are of a `Doctor` object and a `Patient` object.